src/Makefile.qt.include

```diff
@@ -312,6 +312,7 @@ QML_RES_QML = \
   qml/components/StorageLocations.qml \
   qml/components/StorageOptions.qml \
   qml/components/StorageSettings.qml \
+  qml/controls/BlockClock.qml \
```

Comment:
this is a component. not a control

Reply:
The next update of my PR intends to make the `BlockClock.qml` file a purely abstract structure and introduce a component file that uses it to paint different states of the block clock as needed. So I think moving `BlockClock.qml` to components now would be a wasted effort.

Should you still think I shall move the `BlockClock.qml` to controls, or if there is a better approach for achieving the final BlockClock state, I would love to hear that.